centreon/www/front_src/src/Main/index.test.tsx

```diff
@@ -294,7 +294,8 @@ describe('Main', () => {
     window.history.pushState({}, '', '/');
   });
 
-  it('displays the login page when the path is "/login" and the user is not connected', async () => {
+  // To migrate to Cypress
+  it.only('displays the login page when the path is "/login" and the user is not connected', async () => {
```

Comment:
why `only` is used ?
other tests are not compatible with jotai v2 and must be migrated to cypress ?

Reply:
Yes they have to be migrate to Cypress, I have created a ticket for this specific thing.